spec/lib/api/redfish-1.0/update-service-spec.js

```diff
@@ -237,6 +394,17 @@ describe('Redfish Update Service', function () {
             });
     });
 
+    it('should handle error while rendering result', function() {
+        redfish.render.restore();
+        this.sandbox.stub(redfish, 'render').rejects('test');
+        return helper.request().get('/redfish/v1/UpdateService')
+            .expect('Content-Type', /^application\/json/)
+            .expect(400)
+            .expect(function (response) {
+                expect(response.body).to.have.property('message', 'test');
+            });
+    });
+
```

Comment:
It' better to add case for success

Reply:
Other cases had been covered. This case is specified for the error case.